volplugin/init.go

```diff
@@ -0,0 +1,92 @@
+package volplugin
+
+import (
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"github.com/contiv/errored"
+	"github.com/contiv/volplugin/config"
+	"github.com/contiv/volplugin/lock"
+	"github.com/contiv/volplugin/storage/backend"
+	"github.com/docker/engine-api/client"
+	"github.com/docker/engine-api/types"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func (dc *DaemonConfig) updateMounts() error {
+	dockerClient, err := client.NewEnvClient()
+	if err != nil {
+		return errored.Errorf("Could not initiate docker client").Combine(err)
+	}
+
+	containers, err := dockerClient.ContainerList(context.Background(), types.ContainerListOptions{})
```

Comment:
do we need to worry about containers that are down or went down while volplugin was down?